kuma/static/js/analytics.js

```diff
@@ -1,8 +1,20 @@
 (function(win, doc, $) {
     'use strict';
 
+    // prettier-ignore
     // Adding to globally available mdn object
     var analytics = mdn.analytics = {
+        /**
+         * Handles postMessage events from the interactive editor, passing of
+         * the data to `trackEvent` if the origin of the message is what we expect.
+         * @param {Object} event - The event Object received from the postMessage
+         */
+        interactiveExamplesEvent: function(event) {
+            if (event.origin !== 'https://mdn.github.io') {
+                return false;
+            }
+            this.trackEvent(event.data);
```

Comment:
I had a closer look at the MDN code and I think this is okay after all.

I though the MDN code was like this

```
var data = {
                hitType: 'event',
                eventCategory: eventObject.category || '',    // Required.
                eventAction: eventObject.action || '',             // Required.
                eventLabel: eventObject.label || '',
                eventValue: eventObject.value || 0,
                hitCallback: eventObject.callback || null
            };
```

but it's like like this:

```
var data = {
                hitType: 'event',
                eventCategory: eventObject.category || '',    // Required.
                eventAction: eventObject.action || '',             // Required.
                eventLabel: eventObject.label || '',
                eventValue: eventObject.value || 0,
                hitCallback: callback || null
            };
```